.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+#### What
+
+#### Ticket
+
+[board ticket description](link)
+
+#### Why
+
+#### How
+
+--------
+
+#### TODO (wip)
+
+ - [ ] item 1
+ - [X] item 2
```

Comment:
What is the likelyhood of you actually using this section? If this is something you will not use in the short term and adds noise the the PR i would remove for now and revisit when there maybe actual PR items you would want to check off

Reply:
Fair enough. That's a good point